### PR TITLE
add developement requirements

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/hosting_guide/requirements.rst [hunk 1]
```diff
@@ -1,14 +1,26 @@
 Requirements
 ============
 
-Webserver
----------
+Front servers
+-------------
+
```
Comment (itkg-rboessel) [inline en/hosting_guide/requirements.rst]: What is the represented by Front servers ?

File: en/hosting_guide/requirements.rst [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+Production server
+~~~~~~~~~~~~~~~~~
 
 It's required to have a PHP webserver in order to run Open Orchestra.
 
 * PHP 5.4+
 * Apache 2 or NginX
 * Varnish 3 reverse proxy (optionnal)
+* Nodejs
```
Comment (ngilain) [inline en/hosting_guide/requirements.rst]: As it is a doc on production server, I think you should mention that Open Orchestra is "certified" on certain distrib as Debian.

Comment (ngilain) [inline en/hosting_guide/requirements.rst]: You forgot to talk about mongodb

Comment (nicolasThal) [inline en/hosting_guide/requirements.rst]: It talk about the database in the rest of the document

Comment (ngilain) [inline en/hosting_guide/requirements.rst]: Ok then I think the menu is not optimal : what interest a sys admin requested to prepare an open orchestra environnement is to get a short and exhaustive list of what is required and a second one with what is optional. Two short bullet list.

The menu you're proposing is more a functional one and corresponds to an explanation of who does what. That does not matter to a sys admin. He only want to know what tool on which version.

Comment (ngilain) [inline en/hosting_guide/requirements.rst]: You should also talk about php extensions such as mongo and imagick

**Comment by ngilain** [comment]
@nicolasThal : I'd see something more like this

File: en/hosting_guide/requirements.rst [hunk 3]
```diff
@@ -1,17 +1,43 @@
 Requirements
 ============
 
-Webserver
----------
+Open Orchestra is developped on a specific environnement, making that environnement our reference.
+Open Orchestra is certified and supported on that reference environnement. This document provides
+you the different technologies required to reproduce that reference environnement.
```
Comment (juchi) [inline en/hosting_guide/requirements.rst]: environment

**Comment by nicolasThal** [comment]
@itkg-ngilain I agree with you :)

**Comment by ngilain** [comment]
@juchi : typo fixed
@nicolasThal : nice :) before merging, can you help me finding the versions we're using on the VM for these softs : NodeJs, Blackfire and Selenium ?

**Comment by juchi** [comment]
Actually it's "environment" not "environement" ;)

**Comment by nicolasThal** [comment]
npm --version : 1.4.28
node --version : v0.10.30
selenium : 2.44.0 https://github.com/open-orchestra/open-orchestra-provision/blob/master/roles/selenium/vars/main.yml#L4
Blackfire is still experimental, I don't think that the version matter a lot

**Comment by ngilain** [comment]
@juchi : I hope this time is the good one >< If not, I find another word :p
@nicolasThal : thank you for your help, are you ok with that version of the doc ?

**Comment by nicolasThal** [comment]
:+1:

**Comment by juchi** [comment]
:+1:

**Pull request merged**